### ISSUE TITLE
Indexing not working and cannot add a druid to existing collection

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/exhibits#2265]
Let me know if these should be separate tickets (indexing and not being able to add druid). 

Failed to add druid: When adding the druid, I added the druid to the end of the list of druids under Curation: Add items page. Then I click Add items at the bottom. I go to "Item status" and enter the druid and it says "no matches found". 

Failed to reindex: Here is a screenshot of what I get when I try to index: 

<img width="872" alt="Screen Shot 2023-01-11 at 5 52 44 PM" src="https://user-images.githubusercontent.com/3269689/212137629-016b2b58-d951-4782-a490-b122130494b6.png">

When I click on "failed" this report comes up:

`Spotlight::ReindexExhibitJob a4b94bb6
×
Failed about 3 hours ago in less than a minute

Message Details
1 job started January 11, 2023 14:37
2 job created by [lsorens@stanford.edu](mailto:lsorens@stanford.edu)
3 wrong number of arguments (given 1, expected 0)
{
"backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'", "resource_id": 25269 } 4	wrong number of arguments (given 1, expected 0)	 { "backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'",
"resource_id": 25266
}
5 wrong number of arguments (given 1, expected 0)
{
"backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'", "resource_id": 25270 } 6	wrong number of arguments (given 1, expected 0)	 { "backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'",
"resource_id": 25270
}
7 wrong number of arguments (given 1, expected 0)
{
"backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'", "resource_id": 25267 } 8	wrong number of arguments (given 1, expected 0)	 { "backtrace": "/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/activesupport-7.0.4/lib/active_support/core_ext/object/blank.rb:18:in blank?'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in reject'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:105:in convert_stored_value_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:68:in block in custom_fields_data_to_solr'\n/opt/app/exhibits/exhibits/shared/bundle/ruby/3.1.0/gems/blacklight-spotlight-3.5.0.2/app/models/spotlight/solr_document_sidecar.rb:64:in each'",
"resource_id": 25268
}
9 processed 0 / 6
10 job failed January 11, 2023 14:37`

**Comment by jcoyne** [comment]
Honeybadger exception: https://app.honeybadger.io/projects/49092/faults/92403183

**Comment by marlo-longley** [comment]
@laurensorensen which druid are you trying to add?

**Comment by laurensorensen** [comment]
@marlo-longley it's yt594yv4496 
Thanks!

**Comment by cbeer** [comment]
There's something really weird and confusing about the relationship between virtual-tribunals exhibit and laurens-vt-test. They've ended up sharing exhibit updated items, and some of those items in laurens-vt-test were broken (using symbols instead of strings for keys).

I've remediated those 5 objects (the only ones like it in the whole database), which should unblock indexing.

**Comment by laurensorensen** [comment]
Hi @cbeer for some reason I can't re-index. We saw an issue [where on this page ](https://exhibits.stanford.edu/virtual-tribunals/feature/imt-crimes)the a bunch of these boxes (scroll down on the page) are showing "0 items" when there should be many there. I tried to re-index and it failed. Putting this here in case it's the same issue - if not let me know and I'll make a new ticket. 
<img width="824" alt="Screen Shot 2023-01-20 at 12 40 23 PM" src="https://user-images.githubusercontent.com/3269689/213801754-abb05ce7-f53c-4ef2-bdaa-c89beea77235.png">

**Comment by caaster** [comment]
Hi Access Team, if this is the same problem -- what can be done to further troubleshoot this so it stops happening? Indexing has failed at least 2 or 3 times over the past few months. Thank you.

**Comment by cbeer** [comment]
@laurensorensen It looks like the searches on the feature page you linked facet on the collection title "International Court of Justice Archives of the Nuremberg International Military Tribunal", but that collection was later renamed "Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945-46"?

**Comment by laurensorensen** [comment]
Hi all, I re-ran the indexing and it was successful this time. Not sure what the issue was before.

**Comment by laurensorensen** [comment]
@cbeer Weird! I'm not seeing that. It's "Taube Archive..." everywhere I can see. Can you post a screenshot?

**Comment by laurensorensen** [comment]
@cbeer Oh wow thank you for noticing that! I guess the way to fix is to remediate the name of the collection in the saved search? I will try and figure out how that's done.

**Comment by caaster** [comment]
@laurensorensen please reach out to me if you have questions. If it were me, I'd create a new search, and make sure to correct the link(s) on the feature page with the new saved search.

**Comment by caaster** [comment]
@laurensorensen can this ticket be closed now?

**Comment by laurensorensen** [comment]
Yes, closing now.
